src/components/react-admin/DraggableFormIterator.js

```diff
@@ -33,10 +32,14 @@ import { CSSTransition, TransitionGroup } from 'react-transition-group';
 import compose from '../../utils/compose';
 import './draggable.scss';
 
-const DragHandle = sortableHandle(() => <DragHandleIcon />);
+const DragHandle = sortableHandle(({ classes }) => (
+  <span className={classes.dragHandle}>
+    <DragHandleIcon />
+  </span>
+));
```

Comment:
Maybe, ou can render this component dumber :
```
const DragHandle = sortableHandle(props => (
  <span {...props}>
    <DragHandleIcon />
  </span>
));
```

And its call below
```
<DragHandle className={classes.dragHandle} />
```

Reply:
done